docs/backup-restore-integration.md

```diff
@@ -0,0 +1,334 @@
+# Backing Up and Restoring KubeVirt resources
+
+## Introduction
+
+This document is intended for developers building backup/disaster recovery solutions for KubeVirt.
+
+Such solutions typically perform the following high level operations.
+
+### Backup
+
+1.  Build dependency graph for all required Kubernetes resources
+2.  Quiesce (freeze) applications
+3.  Snapshot PersistentVolumeClaim data
+4.  Unquiesce (thaw) applications
+5.  Copy all necessary Kubernetes resource definitions to a shared storage location
+6.  (optional) Export snapshotted PVC data to a shared storage location
+
+Steps 3, 5, and 6 are beyond the scope of this document.
+
+### Restore
+
+1.  Populate PersistentVolumeClaims with snapshot data
```

Comment:
Should we be more specific here and add details about our expectations around restored PVs with initialized claimRefs?

Reply:
For in cluster backup/restore I don't think we can assume that there will always be prepopulated PVs with claimrefs set.  The prePopulated/populatedFor annotations applied to DataVolumes/PVCs will help us deal with that uncertainty.